documentation/anonymization.markdown

```diff
@@ -1,79 +1,75 @@
+The tool walks through your entire data set, applying different rules to each of the leaf data based on the profile you have chosen. The data structure is left intact, and only the data contents are modified. This maintains the "shape" of the data without exposing the underlying data you may wish to scrub. 
 
-By keeping the shape the data can be anonymized based on your needs but still hold some value to the consumer of your anonymized data.
+We do this by always ensuring we replace data consistently. For example, if a string is replaced with something random, we ensure that all instances of that original string are replaced with the same random string throughout the data.
 
-Anonymization Profiles
+By keeping its original shape, the data can be anonymized based on your needs but still hold some value to the consumer of your anonymized data.
+
+Anonymization profiles
 -----
 
-Anonymizing all data is often not that useful so we have provided you with a number of different profiles that you can choose from that will provide different levels of anonymization.
+You may not need to anonymize all data in every case, so we have provided a number of profiles offering varying levels of anonymization.
 
-The profile itself can be chosen on the command line when the command is run. For example, you can choose the `low` profile as an option like so:
+The profile can be specified on the command line when the command is run. For example, to choose the `low` profile, enter:
 
     $ sudo puppetdb export --outfile ./my-puppetdb-anonymized-export.tar.gz --profile low
 
 ### Profile: full
 
-The `full` profile will anonymize everything, while keeping the shape of data as previously mentioned. This includes: node names, resource types, resource titles, parameter names, values, any log messages, file names and file lines. The result should be a completely anonymized data set. Report metrics under the `resources` and `events` categories are left intact, as these can be inferred from the rest of the data, but names of metrics under the `time` category are anonymized as resource types.
+The `full` profile will anonymize all data (including node names, resource types, resource titles, parameter names, values, any log messages, file names, and file lines) while retaining the data set's shape. The result is a completely anonymized data set. Report metrics under the `resources` and `events` categories are left intact, as these can be inferred from the rest of the data, but names of metrics under the `time` category are anonymized as resource types.
```

Comment:
Oxford comma is here to stay! :P https://confluence.puppetlabs.com/display/Writing/Style+and+formatting